examples/advanced/main.tf

```diff
@@ -175,6 +212,34 @@ module "ocp_base" {
     instance_id = module.kp_all_inclusive.kms_guid
     crk_id      = module.kp_all_inclusive.keys["${local.key_ring}.${local.cluster_key}"].key_id
   }
+  cbr_rules = [
+    {
+      description      = "${var.prefix}-event notification access only from vpc"
+      enforcement_mode = "enabled"
+      account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+      rule_contexts = [{
+        attributes = [
+          {
+            "name" : "endpointType",
+            "value" : "public"
+          },
+          {
+            name  = "networkZoneId"
+            value = module.cbr_vpc_zone.zone_id
+        }]
+        }, {
+        attributes = [
+          {
+            "name" : "endpointType",
+            "value" : "public"
+          },
+          {
+            name  = "networkZoneId"
+            value = module.cbr_zone_schematics.zone_id
+        }]
```

Comment:
Same code needs to be added for the fscloud example (schematics zone).

Reply:
Also to apply correct CBR rule here, you ned to pass `operations` attribute value in CBR rule.

```
operations = [{
    api_types = [
      {
        "api_type_id": "crn:v1:bluemix:public:containers-kubernetes::::api-type:management"
      }
    ]
  }]
```